pkg/util/test/util.go

```diff
@@ -776,6 +781,12 @@ func ValidateUninstallStorageCluster(
 		return err
 	}
 
+	// Verify telemetry secret is deleted on UninstallAndWipe
```

Comment:
can we check the delete strategy to be UninstallAndWipe first then validate the secret deletion?

Reply:
In this func we only validate after the wipe, cluster and all components are gone at this point, the func that uninstalls cluster is setting the UninstallAndWipe strategy